apps/evans/components/pages/document/ListenBlock.tsx

```diff
@@ -7,19 +7,22 @@ import DownloadAudiobook from './DownloadAudiobook';
 import { LANG } from '@/lib/env';
 import EmbeddedAudio from '@/components/core/EmbeddedAudio';
 
-export interface Props {
+export type AudioPart = {
   title: string;
+  url: string;
```

Comment:
would it be helpful for the api to tell us the duration in seconds as well? would that prevent the `NaN:NaN` initial load thing i'm seeing now?

Reply:
Maybe that would be nice...